docs/scripts/buildApi.ts

```diff
@@ -1295,14 +1295,33 @@ Page.getInitialProps = () => {
   }
 }
 
-function run(argv: { componentDirectories?: string[]; grep?: string; outputDirectory?: string }) {
+/**
+ * Creates .js file containing all /api nextjs pages
+ */
+function generateApiPagesManifest(outputPath: string, prettierConfigPath: string): void {
+  const [{ children: apiPages }] = findPages({ front: true });
```

Comment:
`findPages({ front: true })` was what we pre-evaled previously in `findPage.js`.